FirebaseRemoteConfigSwift/Sources/PropertyWrapper/RemoteConfigValueObservable.swift

```diff
@@ -61,13 +55,19 @@ internal class RemoteConfigValueObservable<T: Decodable>: ObservableObject {
     if FirebaseApp.app()?.name != appName {
       return
     }
+    applyConfigValue()
+  }
+
+  private func applyConfigValue() {
     do {
       let configValue: RemoteConfigValue = remoteConfig[key]
-      if configValue.source == .remote {
+      if configValue.source == .remote || configValue.source == .default {
         self.configValue = try remoteConfig[key].decoded()
+      } else {
+        self.configValue = fallbackValue
```

Comment:
Is this the codepath or when `source == .static`?

Reply:
This is for `source == .static`, yes. Would you rather I make this  more explicit?

Reply:
I think it might be a good idea to make it more explicit (and thus, easier to understand).